src/store/index.js

```diff
@@ -0,0 +1,100 @@
+import { readUserList, readUserTodoItems, readUser } from '../endpoint/service.js';
+import { ERROR } from '../constants/messageAPI.js';
+const store = {
+  userList: [],
+  user: undefined,
+};
+/* store 를 변경하는 함수는 set 으로 시작됩니다. */
+export const setter = {
+  async userList () {
+    try {
+      store.userList = await readUserList();
+    }
+    catch (err) {
+      alert(err.message);
+    }
+  },
```

Comment:
```suggestion
  userList () {
      readUserList().then(userList => {
        store.userList = userList;
      }).catch(alert);
  },
```
async/await으로 값을 할당하면 proimse 형태로 들어갈 수 있기 때문에 이렇게 해주는게 더 좋을 것 같아요!